recipe/meta.yaml

```diff
@@ -1,34 +1,41 @@
-{% set name = "typeguard" %}
 {% set version = "4.1.5" %}
+{% set min_python = "3.8" %}
+{% set min_typing_extensions = "4.7.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: typeguard
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/typeguard/typeguard-{{ version }}.tar.gz
   sha256: ea0a113bbc111bcffc90789ebb215625c963411f7096a7e9062d4e4630c155fd
 
 build:
   noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 2
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   host:
-    - python >=3.8
+    - python >={{ min_python }}
```

Comment:
use the minimum supported python

Reply:
Weren't we using it before?

Reply:
Yes, but it was hard coded. In the future, when the minimum python version is updated, you will only need to change it once at the top of this recipe instead of at 3 separate places in the recipe.